site/content/en/docs/Getting Started/create-gameserver.md

```diff
@@ -134,7 +134,7 @@ You might also be interested to see the `Events` section, which outlines when va
 Let's retrieve the IP address and the allocated port of your Game Server :
 
 ```
-kubectl get gs -o=custom-columns=NAME:.metadata.name,STATUS:.status.state,IP:.status.address,PORT:.status.ports
+kubectl get gs
```

Comment:
Lots of outputs on this page are the old output :cry: 
See: https://107091d-dot-preview-dot-agones-images.appspot.com/site/docs/getting-started/create-gameserver/